packages/app/src/app/overmind/namespaces/workspace/actions.ts

```diff
@@ -92,10 +92,12 @@ export const sandboxInfoUpdated: AsyncAction = withOwnedSandbox(
   async ({ state, effects, actions }) => {
     const sandbox = state.editor.currentSandbox;
     const { project } = state.workspace;
-
-    const hasChangedTitle = project.title && sandbox.title !== project.title;
+    const hasChangedTitle =
+      project.title && project.title.trim() && sandbox.title !== project.title;
```

Comment:
`project.title && project.title.trim() && ...` is a bit redundant, `project.title.trim() && ...` should be enough.

Reply:
Hey @lbogdan 
thanks for the quick review it does make sense as it will always be a string. I have updated the check.

Reply:
Hopefully it's always a string! @CompuIves should know better. 🙂